src/modelinstance.cpp

```diff
@@ -860,6 +902,18 @@ void ModelInstance::configureBatchSize(const ModelConfig& config, const DynamicM
     }
 }
 
+void ModelInstance::loadTensorFactories() {
+    using std::make_shared;
+    this->tensorFactories.emplace(OVMS_BUFFERTYPE_CPU, make_shared<RegularOVTensorFactory>());
```

Comment:
Emplace:  If the key already exists, the insertion is skipped. Is this expected behavior in this case?